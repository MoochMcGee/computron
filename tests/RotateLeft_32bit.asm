%include "32bit.inc"

mov eax, 0x11223344
rol eax, 4

mov ebx, 0x55667788
rol ebx, 4

mov ecx, 0x99aabbcc
rol ecx, 4

mov edx, 0xddeeff00
rol edx, 4

mov ebp, 0x11223344
rol ebp, 12

mov esp, 0x55667788
rol esp, 12

mov esi, 0x99aabbcc
rol esi, 12

mov edi, 0xddeeff00
rol edi, 12

db 0xf1
