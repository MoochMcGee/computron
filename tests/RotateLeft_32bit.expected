(  config) Loading tmp.bin at 0x00010000
(  screen) No keymap to load!
(    init) BusMouse present
(    init) FDC present
(    init) IDE present
(    init) Keyboard present
(    init) PIC present
(    init) PIC present
(    init) PIT present
(    init) VGA present
(    init) VomCtl present
(    init) CMOS present
(    init) PS2 present
(     cpu) Protected mode Enable = 1
(     cpu) 0xF1: Secret shutdown command received!
(    dump) EAX: 12233441
(    dump) EBX: 56677885
(    dump) ECX: 9AABBCC9
(    dump) EDX: DEEFF00D
(    dump) EBP: 23344112
(    dump) ESP: 67788556
(    dump) ESI: ABBCC99A
(    dump) EDI: EFF00DDE
(    dump) CS: 1000 {00000000:00000}
(    dump) DS: 0000 {00000000:00000}
(    dump) ES: 0000 {00000000:00000}
(    dump) SS: 0000 {00000000:00000}
(    dump) FS: 0000 {00000000:00000}
(    dump) GS: 0000 {00000000:00000}
(    dump) CR0: 00000001
(    dump) A20: 0
(    dump) GDTR: {base=00000000, limit=0000}
(    dump) LDTR: {base=00000000, limit=0000}
(    dump) IDTR: {base=00000000, limit=0000}
(    dump) C=0 P=0 A=0 Z=0 S=0 I=1 D=0 O=0
(    dump)   -  (66 B801000000)
(    dump) 1000:00000049 F1             ??? 
